tests/unittests/onnxImporterTest.cpp

```diff
@@ -235,3 +235,36 @@ TEST(onnx, importDotProduct) {
   auto *batchedReduceAdd = llvm::cast<BatchedReduceAddNode>(saveInput);
   ASSERT_TRUE(llvm::isa<MulNode>(batchedReduceAdd->getBatch()));
 }
+
+/// Test loading Sum with one input and one output
+TEST(onnx, importSum1) {
+  ExecutionEngine EE{BackendKind::Interpreter};
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+  std::string netFilename("tests/models/onnxModels/sum1.onnxtxt");
+
+  Context ctx;
+  Placeholder *output;
+  {
+    Tensor x(ElemKind::FloatTy, {3});
+    x.getHandle() = {1, 2, 3};
+    ONNXModelLoader onnxLD(netFilename, {"x"}, {&x.getType()}, *F);
+    output = onnxLD.getSingleOutput();
+
+    ctx.allocate(mod.getPlaceholders());
+    updateInputPlaceholdersByName(ctx, &mod, {"x"}, {&x});
```

Comment:
@bertmaher I copy-pasted the unit test but what do above 2 lines do?

Reply:
They let us provide the actual input values for the net.  When we load the net the inputs are represented by "placeholders" that aren't backed by any storage.  `ctx.allocate` creates storage for them, and `updateInputPlaceholdersByName` fills them with the named tensor values.